src/SIM/NonLinSIM.C

```diff
@@ -190,6 +196,9 @@ ConvStatus NonLinSIM::solveStep (TimeStep& param, SolutionMode mode,
   if (solution.empty())
     return FAILURE;
 
+  if (rTolSteps > 0 && param.step == 1)
+    rTol *= std::pow(rTolStep, -rTolSteps);
```

Comment:
At least, consider moving this statement into the parse-method (lines 114-117) then no need to check param.step == 1